<launch>
  <arg name="model_name" default="mobile_base"/>
  <arg name="joint_name" default="wheel_left_joint"/>
  <arg name="event_mode" default="timer" />
  <arg name="gazebo_ns" default="/gazebo" />
  <arg name="gazebo_gui_ns" default="/gazebo_gui" />
  <node pkg="gazebo_domain_randomizer" name="joint_properties_randomizer" type="joint_properties_randomizer.py" args="-m $(arg model_name) -j $(arg joint_name) -e $(arg event_mode) --gazebo_ns $(arg gazebo_ns)" output="screen" />
  <node pkg="gazebo_domain_randomizer" name="physics_properties_randomizer" type="physics_properties_randomizer.py" args="-e $(arg event_mode) --gazebo_ns $(arg gazebo_ns)" output="screen" />
</launch>
